app/lib/common/pages/fatal_fail.dart

```diff
@@ -49,28 +50,14 @@ class _FatalFailPageState extends ConsumerState<FatalFailPage> {
                   width: height,
                   child: SvgPicture.asset('assets/images/genericError.svg'),
                 ),
-                const Text(
-                  'Something went terribly wrong:',
-                ),
+                const Text('Something went terribly wrong:'),
```

Comment:
we probably want to translate that...?!?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1580/commits/62c32a1b0b58b2e8d4d16a3bb1de769e87ab4466